src/HTML/makeHtml.go

```diff
@@ -39,13 +39,12 @@ func GenerateHtml(csvData string) {
 			</p>
 			</div>
 			<div class="all-quadrants-mobile show-all-quadrants-mobile"></div>
-			<div class="graph-footer"></div>
+			<div class="graph-footer">
+				<p class="agree-terms">Visit the Novo Nordisk <a href="https://github.com/NovoNordisk-OpenSource/decentralized-tech-radar">Tech Radar open source on Github</a>. This tech radar was heavily inspired by Thoughtworks.</p>
```

Comment:
Is it okay to not link to Thoughtworks?

Reply:
It's ok yes, but I think you're right - we should show more appreciation and link directly to them. That being said, the message might change after Lasse has looked at it, so we can change it at that time.

Reply:
I will write a message @ him on discord and mark this as resolved